src/main/java/competition/electrical_contract/ElectricalContract.java

```diff
@@ -15,5 +15,13 @@ public abstract class ElectricalContract {
 
     public abstract DeviceInfo getSteeringEncoder(SwerveInstance swerveInstance);
 
+    public abstract DeviceInfo getLowerArmLeftMotor();
+
+    public abstract DeviceInfo getLowerArmRightMotor();
+    public abstract DeviceInfo getUpperArmLeftMotor();
+    public abstract  DeviceInfo getUpperArmRightMotor();
+    public abstract  boolean isArmReady();
```

Comment:
Please split this into `isUpperArmReady` and `isLowerArmReady` and make the relevant changes in the subsystems.

Reply:
Changes were made